lib/internal/process/stdio.js

```diff
@@ -91,8 +89,12 @@ function setupStdio() {
         break;
 
       default:
-        // Probably an error on in uv_guess_handle()
-        throw new ERR_UNKNOWN_STDIN_TYPE();
+        // Provide a dummy contentless input for e.g. non-console
+        // Windows applications.
+        const { Readable } = require('stream');
+        stdin = new Readable({ read() {} });
+        stdin.push(null);
+        break;
```

Comment:
Nit: can we remove this `break` and the one below?

Reply:
@lpinca done!